# Message with the task finished

int8 task_id